projects/go-lib/src/lib/components/go-toast/go-toast.component.scss

```diff
@@ -10,10 +10,22 @@
 
   &__actions {
     align-items: center;
+    align-self: flex-start;
     display: flex;
     flex: 1;
     justify-content: flex-end;
-    padding: .875rem 1.25rem;
+    padding: 1.25rem;
```

Comment:
Would it be a good idea to make variables for some of these numeric values?

Reply:
Sure. I think that's a fair point here. I did this with the padding because we use the same `1.25rem` value in multiple spots.